RELEASE.md

```diff
@@ -0,0 +1,34 @@
+# Release
+
+This project produces GitHub Releases as the versioned artifacts for use in products like the Terraform VS Code Extension.
+
+For example, you can download the v0.2.3 Terraform Syntax release by using `
+https://github.com/hashicorp/syntax/releases/download/v0.2.3/terraform.tmGrammar.json`
+
+## Publish Release
+
+## Increment Version
+
+1. Ensure that CHANGELOG.MD has all changes since last release. Add if any are missing.
```

Comment:
Given that you decided to generate the changelog via `gh`, is this still accurate?